storm-client/src/jvm/org/apache/storm/daemon/metrics/SpoutThrottlingMetrics.java

```diff
@@ -26,32 +27,20 @@
     private final CountMetric skippedInactive = new CountMetric();
 
     public SpoutThrottlingMetrics() {
-        this.metricMap.put("skipped-max-spout", skippedMaxSpout);
-        this.metricMap.put("skipped-throttle", skippedThrottle);
-        this.metricMap.put("skipped-inactive", skippedInactive);
-    }
-
-    public CountMetric getSkippedMaxSpout() {
-        return skippedMaxSpout;
-    }
-
-    public CountMetric getSkippedThrottle() {
-        return skippedThrottle;
-    }
-
-    public CountMetric getSkippedInactive() {
-        return skippedInactive;
+        metricMap.put("skipped-max-spout-ms", skippedMaxSpout);
+        metricMap.put("skipped-throttle-ms", skippedThrottle);
+        metricMap.put("skipped-inactive-ms", skippedInactive);
     }
 
-    public void skippedMaxSpout(CommonStats stats) {
-        this.skippedMaxSpout.incrBy(stats.getRate());
+    public void skippedMaxSpoutMs(long ms) {
+        this.skippedMaxSpout.incrBy(ms);
```

Comment:
Why not also rename the fields (skippedMaxSpout, skippedThrottle, skippedInactive) to add `Ms` as postfix?